src/myPyllant/api.py

```diff
@@ -1221,19 +1221,19 @@ async def quick_veto_ambisense_room(
         Parameters:
             room: The target room
             temperature: The target temperature
-            duration_seconds: Optional, sets overwrite for this many seconds
-            default_duration: Optional, falls back to this default duration if duration_seconds is not given
+            duration_minutes: Optional, sets overwrite for this many minutes
+            default_duration: Optional, falls back to this default duration if duration_minutes is not given
         """
         if not default_duration:
             default_duration = (
                 int(DEFAULT_QUICK_VETO_DURATION) * 60
-            )  # duration for quick veto for room is in seconds
+            )  # duration for quick veto for room is in minutes
 
         url = f"{await self.get_api_base()}/api/v1/ambisense/facilities/{room.system_id}/rooms/{room.room_index}/configuration/quick-veto"
 
         payload = {
             "temperatureSetpoint": temperature,
-            "duration": duration_seconds or default_duration,
+            "duration": duration_minutes or default_duration,
```

Comment:
If needed, we can put a guard here. If this is like multimatic, minimum duration is 30 minutes.